bin/node/runtime/src/lib.rs

```diff
@@ -477,6 +477,7 @@ parameter_types! {
 	pub const SlashDeferDuration: pallet_staking::EraIndex = 2;
 	pub const RewardCurve: &'static PiecewiseLinear<'static> = &REWARD_CURVE;
 	pub const MaxNominatorRewardedPerValidator: u32 = 256;
+	pub OffchainRepeat: BlockNumber = 5;
```

Comment:
[Offchain worker fix](https://github.com/paritytech/substrate/pull/8290)

Reply:
Should the value be 5 for us?